misk/src/main/kotlin/misk/prometheus/PrometheusHistogram.kt

```diff
@@ -0,0 +1,18 @@
+package misk.prometheus
+
+import misk.metrics.Histogram
+
+class PrometheusHistogram: Histogram {
+    lateinit var histogram: io.prometheus.client.Histogram
+    var bucketCount: Int = 0
```

Comment:
You need to maintain a reference to the child object in order to get this count.

Reply:
You need to maintain a reference to the child that you made. All access to buckets otherwise seems to be restricted within the Histogram class.

Reply:
We don't seem to use the record function to return anything. Would it make sense to have it return the bucketcount of the child it just made?

Reply:
well what is count even representing? it looks to be scoped to the data point level, which include labels. so having a single count per histogram doesn't even make sense

Reply:
count is representing the number of buckets in a child. A child is the value of a single histogram. It would actually make more sense to return the buckets of a child once you create the child (in record) rather then having a separate method within the prometheus histogram class.